packages/sev-snp-utils-sdk/src/lib/sev-snp-utils-sdk.spec.ts

```diff
@@ -1,13 +1,6 @@
-// @ts-nocheck
-import { TextEncoder, TextDecoder } from 'util';
-import * as fetch from 'node-fetch';
-
-global.TextEncoder = TextEncoder;
-// @ts-ignore
-global.TextDecoder = TextDecoder;
+import fetch from 'cross-fetch';
```

Comment:
Have we tried if this would work happily with js frameworks? eg. nextjs and vite

Reply:
:+1: Updated this to return the same string as our `paramsValidator` does, which mentions the possible strings

Reply:
Yes, this is actually why I replaced our manual polyfill, which was not checking to see if it was already defined and was overwriting the existing version when it shouldn't be -- it was _always_ overwriting the existing `fetch` implementation, in both nextjs and nodejs where fetch is natively supported, even when it was already defined.  `cross-fetch` will only inject `node-fetch` into the environment if `global.fetch` is not already defined.  This version of `cross-fetch` depends on the same version of `node-fetch` that we were manually requiring and always injecting on to `global`, so in cases where `cross-fetch` _does_ inject, it will be the same implementation we previously relied on.